libmscore/glissando.cpp

```diff
@@ -370,6 +380,21 @@ void Glissando::write(Xml& xml) const
       if (_showText && !_text.isEmpty())
             xml.tag("text", _text);
       xml.tag("subtype", int(_glissandoType));
+      writeProperty(xml, P_ID::PLAY_GLISSANDO);
+      switch ( glissandoStyle()) {
+          case MScore::GlissandoStyle::BLACK_KEYS:
```

Comment:
Hi Nicolas, please give me more info on this nameToType/typeToName idea?  It sounds intriguing.